test/visualizer-layout-scale.test.js

```diff
@@ -342,6 +343,14 @@ test('Visualizer layout - scale - calculation height always greater thans longes
     ['1.1601.' + Array(399).fill(1602).map((num, index) => num + index).join('.'), 5]
   ])
 
+  const layoutWithScaleModifier = testLongNodeChains([
+    // Small so scale factor is quite mild and scaledTotal can exceed svgHeight
+    ['1.2', 20],
+    // Just long enough to have an absolute just below the amount that fits in svgHeight
+    ['1.3.' + Array(20).fill(4).map((num, index) => num + index).join('.'), 5]
```

Comment:
I didn't say it last time, but imo this is a really cool way to init an incremental sequence of nums!